src/Movie/Movie.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState, useEffect } from "react";
+import './Movie.css';
+import Search from "./Search/Search";
+import TopBar from "./Top-Bar/Top-Bar";
+import { useNavigate } from "react-router-dom"; 
+
+function Movie() {
+    const [movie, setMovie] = useState({}); 
+    const [searchResults, setSearchResults] = useState([]); 
+    const [error, setError] = useState(null);
+    const [loading, setLoading] = useState(true);
+    const [currentPage, setCurrentPage] = useState(1);
+    const [triggerSearch, setTriggerSearch] = useState(false);
+    const navigate = useNavigate();
+    const itemsPerPage = 12;
+
+    useEffect(() => {
+        const fetchMovieData = async () => {
+            try {
+                const response = await fetch('https://api.themoviedb.org/3/discover/movie?api_key=ad49ccfdb6941afd7ccc12758c1d5ef2');
+                const data = await response.json();
+                setMovie(data);
+                console.log(data);
+            } catch (error) {
+                setError(error.message);
+            }
+            setLoading(false);
+        };
+        fetchMovieData();
+    }, []);
```

Comment:
move function outside useffect so code will be more readable

Reply:
add this key to localstorage auth and login, logout, private routing based on it
`ad49ccfdb6941afd7ccc12758c1d5ef2`